wallets/lnc/client.js

```diff
@@ -26,7 +26,7 @@ async function disconnect (lnc, logger) {
       }, 50)
       logger.info('disconnected')
     } catch (err) {
-      logger.error('failed to disconnect from lnc', err)
+      logger.error('failed to disconnect from lnc: ' + err)
```

Comment:
`logger` no longer supports a variable number of arguments because second argument is now context